.github/workflows/ci.yml

```diff
@@ -12,8 +12,11 @@ jobs:
     strategy:
       matrix:
         platform-arch: [ubuntu-20.04-x64, ubuntu-20.04-arm, macos-13-x64, macos-latest-arm]
-        otp-version: [ 26.2]
+        otp-version: [25.3, 26.2]
         include:
+          - otp-version: 25.3
+            brew-otp-version: 25
+            vscode-publish: true
           - otp-version: 26.2
             brew-otp-version: 26
             vscode-publish: true
```

Comment:
We cannot simultaneously publish 25 and 26. Let's modify the `vscode-publish: true` in the 26 block to `vscode-publish: false`.